src/publicatie_tabellen/save_as_csv.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+import shutil
+
+import django.apps
+from django.conf import settings
+from django.db import connection
+from django.utils.module_loading import import_string
+
+logger = logging.getLogger(__name__)
+
+
+class SaveAsCsv:
+    TMP_DIRECTORY = "/tmp/pgdump"
+
+    def main(self, app_names:list):
+        self.start_dump(app_names)
+        self.upload_to_blob()
+        self.remove_dump()
+        logger.info("Completed DB dump")
+
+    def start_dump(self, app_names:list):
+        os.makedirs(self.TMP_DIRECTORY, exist_ok=True)
+        for app in app_names:
+            for model in django.apps.apps.get_app_config(app).get_models():
+                self._dump_model_to_csv(model)
+
+    def _dump_model_to_csv(self, model):
+        table_name = model._meta.db_table
+        filepath = os.path.join(
+            self.TMP_DIRECTORY, f"{table_name}.csv"
+        )  # filename is model name
+        with open(filepath, "w") as f:
+            select_query = f"SELECT * FROM {table_name}"
+            sql = (
+                f"COPY ({select_query}) TO STDOUT WITH CSV HEADER"
+            )
```

Comment:
This is not dependent on the open file right? For readability and clarity, I would move this outside the with open